packages/web/src/scss/helpers/_links.scss

```diff
@@ -12,6 +12,8 @@
 // Allows link underline everywhere, except for headings
 .link-underlined {
     text-decoration: underline;
```

Comment:
Keep in mind this is a shorthand property for all `text-decoration-*` properties. Now we need to be more explicit with `text-decoration-line` everywhere we work with a visible decoration, mainly in headings. (`text-decoration: none` should be mostly fine.)

![obrazek](https://user-images.githubusercontent.com/5614085/189904099-0b9ec79f-723c-440f-875b-42ababfcf929.png)